src/components/Tutorial/IntroductionSlides.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="tutorial__steps">
+    <section
+      :id="`slide-${index}`"
+      class="slide__step"
+      v-for="(slide, index) in slides"
+      :key="slide.title"
+      :ref="`slide_${index}`"
+    >
+      <h2 class="subheading">{{ slide.title }}</h2>
+      <p class="slide__text">
+        {{ slide.text }}
+      </p>
+      <div v-if="slide.img" class="slide__image">
+        <img
+          :src="slide.img.url"
+          class="tutorial__step-image"
+          :alt="slide.img.caption"
+        />
+      </div>
+      <button
+        class="btn-game--primary-solid btn-full-size"
+        @click="scrollToNextSlide(index + 1)"
+        v-if="index < slides.length - 1"
+      >
+        {{ BUTTON_COPY[index % 5] }}
+      </button>
+      <button
+        class="btn-game--primary-solid btn-full-size"
+        @click="$emit('nextStep')"
+        v-else
+      >
+        Ready to train?
+      </button>
+    </section>
+  </div>
+</template>
+
+<script>
+const BUTTON_COPY = [
+  "Got it!",
+  "Tell me more!",
+  "I'm with you so far!",
+  "Next!",
+  "Moving on!",
+  "Ok!",
+];
+
+export default {
+  setup() {
+    return { BUTTON_COPY };
+  },
+  props: {
+    slides: {
+      type: Object,
+      required: true,
+    },
+  },
+  emits: ["nextStep"],
+  methods: {
+    scrollToNextSlide(nextEl) {
+      let el = document.querySelector(`#slide-${nextEl}`);
+      el.scrollIntoView({ behavior: "smooth" });
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.tutorial__steps {
+  position: relative;
+  overflow-y: scroll;
+  scroll-snap-type: y mandatory;
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
```

Comment:
Wondering if something like PostCSS would handle these ones?

Reply:
i decided to remove it since we weren't supporting ie10 in any case.